pandas/io/json.py

```diff
@@ -75,14 +96,19 @@ def _format_axes(self):
         raise AbstractMethodError(self)
 
     def write(self):
-        return dumps(
+        serialized = dumps(
```

Comment:
then ``.write()`` in the ``Writer`` class is the same and you just call the super e.g.

```
class JsonTableWriter(FrameWriter):

    def writer(self):
         serialized = super(JsonTableWriter, self).writer()
         scheme = to_json_scheme(....)
      return '{{"schema": {}, "data": {}}}'.format(
                 json.dumps(self.schema), serialized)
 ```